src/components/ManagerPage/ManagerApp.js

```diff
@@ -3,50 +3,65 @@
 import React from 'react'
 import styled from 'styled-components'
 
-import Box, { Tabbable } from 'components/base/Box'
+import Box from 'components/base/Box'
 import Text from 'components/base/Text'
+import Button from 'components/base/Button'
 
 const Container = styled(Box).attrs({
   align: 'center',
-  justify: 'center',
-  m: 2,
+  m: 3,
+  p: 4,
+  boxShadow: 0,
+  flow: 3,
 })`
-  width: 150px;
-  height: 150px;
+  width: 156px;
+  height: 186px;
   background: white;
+  line-height: normal;
 `
 
-// https://api.ledgerwallet.com/update/assets/icons/bitcoin
-
-const ActionBtn = styled(Tabbable).attrs({
-  fontSize: 3,
-})``
-
 const AppIcon = styled.img`
   display: block;
   width: 50px;
   height: 50px;
 `
 
+const AppName = styled(Box).attrs({
+  ff: 'Museo Sans|Regular',
+  fontSize: 4,
+  color: 'dark',
+})`
+  display: block;
+  width: 115px;
+  text-align: center;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+`
+
 type Props = {
   name: string,
+  version: string,
   icon: string,
   onInstall: Function,
-  onUninstall: Function,
+  // onUninstall: Function,
```

Comment:
ô ô

Reply:
yeah.. we only have "Install" button now, but I'm pretty sure the uninstall will come in the future in the designs :) so I keep that here